src/rssignores.cpp

```diff
@@ -43,6 +46,20 @@ void RssIgnores::handle_action(const std::string& action,
 					ignore_expr,
 					m->get_parse_error()));
 		}
+
+		int prefix_len = REGEX_PREFIX.length();
```

Comment:
This can be marked `const`. Please do the same for the rest of variables that you declared; it looks like about half of them are never modified and can be marked `const` ;)

Reply:
done, sorry, I'm not used to add const to everything anymore, if you know what I mean